arangod/RocksDBEngine/RocksDBMethods.h

```diff
@@ -185,6 +190,9 @@ class RocksDBBatchedMethods final : public RocksDBMethods {
   rocksdb::Status RollbackToSavePoint() override {
     return rocksdb::Status::OK();
   }
+  rocksdb::Status RollbackToWriteBatchSavePoint() override {
+    return rocksdb::Status::OK();
+  }
   void PopSavePoint() override {}
```

Comment:
Is it a good idea to have these methods just do nothing, rather than throwing a not implemented error?

Reply:
See above.